src/plugins/discover/public/application/components/default_discover_table/default_discover_table.tsx

```diff
@@ -214,7 +222,7 @@ const DefaultDiscoverTableUI = ({
     }
 
     return () => cancelAnimationFrame(tableLayoutRequestFrameRef.current);
-  }, [columns, tableElement]);
+  }, [columns, tableElement, indexOfRenderedData]);
```

Comment:
should we use `displayedColumnNames` here? that's the actual columns displayed in table.